### ISSUE TITLE
Gradient Accumulation

### ISSUE THREAD

**Issue by ZYX-MLer** [Deci-AI/super-gradients#1858]
### 💡 Your Question

Since I only have one GPU, does the current framework support gradient accumulation training, where gradients are accumulated over multiple forward passes before optimizing parameters?

**Comment by geifmany** [comment]
Hi,
You can see in training params batch accumulate (see [here](https://github.com/Deci-AI/super-gradients/blob/efcac27e1ab440e5b6f7a98954ebcecc9e382066/src/super_gradients/training/params.py#L14)).
Please notice that in very small batches BN stats would not be stable enough, so batch accumulate is not identical to having a large batch.

**Comment by ZYX-MLer** [comment]
thanks, it worked, I will close the issue
